kaeru/views.py

```diff
@@ -41,6 +44,40 @@ def tutorial_view(request):
 def index_view(request):
     return render_to_response('index.html', {})
 
+#todo: uncomment below when user functionality is working
```

Comment:
user functionality is working, you just currently need to enter the signup-url manually.